tests/integration/targets/lookup_hashi_vault/lookup_hashi_vault/tasks/tests.yml

```diff
@@ -73,4 +83,3 @@
           environment:
             VAULT_ADDR: 'https://localhost:8201'
             VAULT_SKIP_VERIFY: False
-      
```

Comment:
nit but, please preserve newline at EOF

Reply:
done

Reply:
removing this line is necessary to pass sanity test:
https://github.com/ansible-collections/community.hashi_vault/pull/50/checks?check_run_id=1825111960

Reply:
I see, that may have been my mistake, I suppose it was originally 2 blank lines (shown in diff view as 1 blank line + EOF), and I misinterpreted it. Apologies!